src/features/jumpNav.tsx

```diff
@@ -189,16 +198,15 @@ const toggleLinkedRefs = () => {
   document.querySelector(".rm-reference-container .rm-caret").scrollIntoView();
 };
 const toggleUnlinkedRefs = () => {
-  (
-    document.querySelector(
-      ".rm-reference-main > div > div:nth-child(2) > div > span > span"
-    ) as HTMLElement
-  ).click();
-  document
-    .querySelector(
-      ".rm-reference-main > div > div:nth-child(2) > div > span > span"
-    )
-    .scrollIntoView();
+  try {
+    const unlinkedRefsCaret = document.querySelector(
+      ".rm-reference-main > div > div:nth-child(2) > div > span"
+    ) as HTMLElement;
+    unlinkedRefsCaret.click();
+    unlinkedRefsCaret.scrollIntoView();
+  } catch (e) {
+    throwErrorToast(e);
+  }
```

Comment:
fix for `Toggle Unlinked Refs`

How do you feel about this format?  I think giving the user a message would improve the UX.

Reply:
If it's acceptable I will apply it to various other commands (specifically the type errors)

![image](https://github.com/dvargas92495/roamjs-workbench/assets/3792666/978bfc73-0805-4484-b7c7-deaa7eeec77c)

Reply:
In general, I'm a big fan of surfacing errors to the user with the toast.

I think however in this case we could move it up the stack. i.e. make it part of `addCommand` instead.

When the method is this scoped, let's just solve the bug directly, by checking `unlinkedRefsCaret` is null

Reply:
"Well, that escalated quickly"
So I made some changes ... and might have gotten carried away 😁

It looked like the current method for finding the caret node was no longer useful:
- Pages without linked references don't show the linked references element at all.  
- Additionally, Discourse Nodes have an extra element, and Discourse Nodes with Canvas references have multiple extra elements.

I then added support for a user activating the command on DNP or a block.

The console log has been left in as I couldn't find the example for emails in samepage.

Let me know your thoughts.

Reply:
Also, I didn't see an `isDNP` or `isBlock` via url in roamjs-components utils.
If I missed it or there is a better way, definitely let me know!

Reply:
> It looked like the current method for finding the caret node was no longer useful:

yea, we'll want to do a better job targeting, likely doing something like `querySelectorAll().find(el => el.innerText.includes("Unlinked References"))`

> I didn't see an isDNP or isBlock via url in roamjs-components utils

Roam provides a way with `window.roamAlphaAPI.mainWindow.mainWindow.getOpenPageOrBlockUid()`. If it's `null`, it's the DNP log, otherwise it's a block/page.

Reply:
re the first point, I just saw your changes that you're already taking that approach, which I think is right